assets/src/edit-story/output/element.js

```diff
@@ -22,8 +27,9 @@ import WithMask from '../masks/output';
 import { getDefinitionForType } from '../elements';
 import { getBox } from '../units/dimensions';
 import WithLink from '../components/link/output';
+import { CTA_ZONE_PERCENT } from '../constants';
 
-function OutputElement({ element }) {
+function OutputElement({ element, cta }) {
```

Comment:
Do we really need to pass `cta` here? The `element.link.type` already has that info, right? Though if you feel strongly about this - that's ok.